src/validations.ts

```diff
@@ -12,8 +13,12 @@ export function validateEmail(
 
   if (!emailRegex.test(value)) {
     return commonErrorKey;
+  } else {
+    const matchAccents = value.match(accentRegex);
+    if (matchAccents !== null && matchAccents[0] !== value) {
```

Comment:
Do you think that reducing nesting is possible?

Something like this:

```suggestion
  } else if (CONDITION) {
```

It will make this code simpler and easier to read.

Reply:
i try use something like this `if(value.match(accentRegex) !== null && value.match(accentRegex) [0] !== value)` but the validator says `value.match(accentRegex) [0]` its possibly null